File: apigateway-openapi-3.0-import/lambda.js
exports.handler = function(event, context, callback) {
	console.log('Received event:', JSON.stringify(event, null, 2));
	let res = {
		"statusCode": 200,
		"headers": {
			"Content-Type": "application/json"
		},
		"body": JSON.stringify({
			"message": "Hello World!"
		})
	};
	callback(null, res);
};

File: apigateway-rest-lambda-api-key/lambda/lambda.py
import json
import requests

def lambda_handler(event, context):
    print(event)
    try:
        ip = requests.get("http://checkip.amazonaws.com/")
    except requests.RequestException as e:
        print(e)
        raise e
    return {
        "statusCode": 200,
        "body": json.dumps({
            "message": "Hello from Lambda backend.",
             "yourIpAddress": ip.text.replace("\n", "")
        }),
    }
